shared/types/form/form.ts

```diff
@@ -60,12 +73,24 @@ export enum FormAuthType {
   SGID_MyInfo = 'SGID_MyInfo',
 }
 
+export enum Language {
+  ENGLISH = 'English',
```

Comment:
Recommend to use [unicode locales](https://www.unicode.org/reports/tr35/#Unicode_locale_identifier) for the values since we might be supporting localization for other countries.
```
ENGLISH = 'en-SG',
CHINESE = 'zh-SG',
  MALAY = 'ms-SG',
  TAMIL = 'ta-SG',
```